beaker.gemspec

```diff
@@ -40,6 +40,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'inifile', '~> 2.0'
   s.add_runtime_dependency 'rsync', '~> 1.0.9'
   s.add_runtime_dependency 'open_uri_redirections', '~> 0.2.1'
+  s.add_runtime_dependency 'in-parallel', '~> 0.1.8'
```

Comment:
Does this have to specify the z component? I think that `0.1` is actually what you want specify, unless you don't plan on revving up to the `0.2.0` version anytime soon...